tests/baselines/reference/downlevelLetConst1.js

```diff
@@ -4,4 +4,3 @@
 const
 
 //// [downlevelLetConst1.js]
-var ;

```

Comment:
Spooky given this test is named in such a way that implies that it's testing exactly this transformation?

Reply:
see https://github.com/microsoft/TypeScript/pull/59893/files#r1749158145 , i think the same principle would apply here

this particular test case is testing `const`, the next one in line is testing `const a` and the next one is `const a = 1`. So they just exercise different kinds of inputs and now the question is if it's ok to drop the variable declaration that was invalid in the first place